test/msgpack.cpp

```diff
@@ -76,7 +76,7 @@ TEST_CASE(test_msgpack_float)
     migraphx::value v = 3.0;
     auto buffer       = migraphx::to_msgpack(v);
     EXPECT(buffer == msgpack_buffer(3.0));
-    EXPECT(migraphx::from_msgpack(buffer) == v);
+    EXPECT(migraphx::from_msgpack(buffer).to<float>() == v.to<float>());
```

Comment:
Why does this need to be converted to `float` first?

Reply:
Function migraphx::from_msgpack returns value which needs to be converted to other types (except for string and bool case) when we are comparing output values. The tests above convert value to appropriate type [(here](https://github.com/ROCmSoftwarePlatform/AMDMIGraphX/blob/bded09498e09b6fbfe93e50e4b23d0208b517de5/test/msgpack.cpp#L58) is the example for int).